admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -158,29 +159,29 @@ object DelegateApiHandler {
     def writeBuf[T](t: T): Buf = Buf.ByteArray.Owned(mapper.writeValueAsBytes(t))
   }
 
+  def getDelegateRsp(dtab: String, path: String, interpreter: NameInterpreter): Future[Response] =
+    (dtab, path) match {
+      case (DtabStr(d), PathStr(p)) =>
+        Delegator(d, p, interpreter).values.toFuture().flatMap(Future.const).map { tree =>
```

Comment:
Comment: io.buoyant.namerd.RichActivity adds a `toFuture` method to Activity.  Unfortunately, we don't want to depend on namerd here.  I wonder if we should create a utility project that we can move RichFuture into...